test/powershell/SDK/PSDebugging.Tests.ps1

```diff
@@ -184,5 +185,94 @@ Describe "PowerShell Command Debugging" -tags "CI" {
 
 }
 
+# Scripting\Debugging\RunspaceDebuggingTests.cs
+Describe "Runspace Debugging API tests" -tag CI {
+    Context "PSStandaloneMonitorRunspaceInfo tests" {
+        BeforeAll {
+            $runspace = [runspacefactory]::CreateRunspace()
+            $runspaceType = [PSMonitorRunspaceType]::WorkflowInlineScript
+            $monitorInfo = [PSStandaloneMonitorRunspaceInfo]::new($runspace)
+            $instanceId = $runspace.InstanceId
+            $parentDebuggerId = [guid]::newguid()
+            $ps = [powershell]::Create()
+            $embeddedRunspaceInfo = [PSEmbeddedMonitorRunspaceInfo]::New($runspace,$runspaceType,$ps, $parentDebuggerId)
+        }
+        AfterAll {
+            $runspace.Dispose()
+            $ps.dispose()
+        }
+        
+        It "PSStandaloneMonitorRunspaceInfo should throw when called with a null constructor" {
```

Comment:
>null constructor [](start = 76, length = 16)

I'm not sure what you mean here - sometimes people incorrectly refer to a default constructor as the null constructor, but that's not what you're testing.